packages/htmlbars-runtime/lib/hooks.js

```diff
@@ -909,9 +914,9 @@ export function getCellOrValue(reference) {
   return reference;
 }
 
-export function component(morph, env, scope, tagName, params, attrs, template, visitor) {
+export function component(morph, env, scope, tagName, params, attrs, template, inverse, visitor) {
```

Comment:
Could we just make this:

```javascript
export function component(morph, env, scope, tagName, params, attrs, templates, visitor) { }
```

Where `templates` is an object that contains `template` and `inverse` properties?  That would in theory align us for future work in Handlebars/HTMLBars for named blocks without changing the component hook signature again...